jupyterlab_nova/handlers.py

```
@@ -53,3 +54,14 @@ def get(self):
     # TODO
     pass
 
+class RuntimeEnvHandler(APIHandler):
+  """Handler to obtain runtime environment"""
+
+  def get(self):
+    command = "cat $ENV_VERSION_FILE_PATH"
+    p = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
```

I would use the native file functions rather than calling out to the shell.

I would do something like this so we can fail gracefully if the environment variable is not set or the file doesn't exit.
```
version = 'unknown'
try:
    env_version = os.environ['ENV_VERSION_FILE_PATH']
    with open(env_version) as f:
        version = f.read().rstrip()
except (KeyError, OSError):
    pass # Could do some logging in here
```